### DISCUSSION TITLE
Adding Step 5: Check FAQs to Quick Start?

### DISCUSSION THREAD

**Discussion by PurpleHullPeas** [aegean-odyssey/mpmd_marlin_1.1.x#74] [General]
I occasionally make edits to the wiki based on questions I see on social media (usually Facebook). I kind of want to add a final step to the Quick Start that basically says "Check the FAQs before posting your question to GitHub/Reddit/Facebook", but I am on the fence. This comes after answering a [fairly common question](https://www.facebook.com/photo?fbid=10218152031941693) the other day, me [adding it to the FAQs](https://github.com/aegean-odyssey/mpmd_marlin_1.1.x/wiki/Frequently-Asked-Questions#windows-10-requires-special-drivers) yesterday, and then someone on Reddit [asking the same question](https://www.reddit.com/r/mpminidelta/comments/mvp4mb/help_pronterface_not_working_i_think_i_got_the/) today.

**Comment by BackwardEcho** [comment]
If you think it's needed, maybe even adding mention of the other MPMD FAQs if you think they are relevant enough to the firmware operation. Though I wonder how often they will be checked... I just know I asked something you know is on a FAQ somewhere, but in my defense I think I just couldn't find some things. Like in the case of the aforementioned issue, I believe I had trouble finding the driver itself. I forget though.

**Comment by rgrobe** [comment]
Codifying knowledge in an FAQ is a noble thing to do, and it should really result in fewer (repeat) questions being asked.
Then again, if any FAQ (or Wikipedia article or forum thread) grows (too) large/rich in information, a lot of people will be overwhelmed by it and will "just ask for a simple solution/explanation".
So it's not just about codifying knowledge but also making it accessible, e.g. by structuring (sections, cross-referencing, ...), by simplyfying it ("101" vs. deep dives) or by reordering it (most common topics first - how many people look at the second or third page of Google results?). This may eventually be solved by AI systems (sigh - there are probably already a number of systems that are smarter than A LOT OF people...)

**Comment by PurpleHullPeas** [comment]
@BackwardEcho I think the mpmd_marlin_1.1.x FAQ should stick to things specific to this firmware and stick to things that have been actual questions about the firmware from people on GitHub/Reddit/Facebook. 

Speaking from experience, @rgrobe is correct that the broader MPMD subject can become a behemoth of a topic that grows from 12 pages on initial release to over 60 pages today. It is best to limit scope when possible.

**Comment by PurpleHullPeas** [comment]
### Step 5: Frequently Asked Questions

Are you having trouble? Before posting a question to GitHub or social media, please check the [FAQs](https://github.com/aegean-odyssey/mpmd_marlin_1.1.x/wiki/Frequently-Asked-Questions) for some of the most common questions related to **mpmd_marlin_1.1.x**. You could also try reading through the topics in the sidebar to see if it is covered elsewhere in this wiki. 

Still cannot find your problem listed? The preferred platform for questions related to **mpmd_marlin_1.1.x** is GitHub. For firmware bugs, you can file an [issue](https://github.com/aegean-odyssey/mpmd_marlin_1.1.x/issues). For general troubleshooting questions, you can start a [discussion](https://github.com/aegean-odyssey/mpmd_marlin_1.1.x/discussions).
